jsf/multi-jsf-installer/src/main/resources/mojarra-deploy.scr

```diff
@@ -1,3 +1,3 @@
-module add --name=javax.faces.api --slot=mojarra-${jsf-version} --resources=jsf-api-${jsf-version}.jar --module-xml=mojarra-api-module.xml
+module add --name=javax.faces.api --slot=mojarra-${jsf-version} --resources=jboss-jsf-api_2.3_spec-${jsf-version}.jar --module-xml=mojarra-api-module.xml
```

Comment:
Same here, this will cause issues when attempting to install Mojarra 2.2.x or Mojarra 2.1.x.